TestGeneration.Extensions.NUnit/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="TestGeneration.Extensions.NUnit.113f61ed-a246-4431-809d-bcd393642488" Version="1.0.0" Language="en-US" Publisher="NUnit.Org" />
+    <Identity Id="TestGeneration.Extensions.NUnit.113f61ed-a246-4431-809d-bcd393642488" Version="1.0.5" Language="en-US" Publisher="NUnit.Org" />
```

Comment:
I just update the last number during fix, and there was 5 steps.  When it releases it should be updated to next valid number,  1.2.0, if we continue the odd/even practice.  Didn't want to update that number yet, would wait until ready for release.